### ISSUE TITLE
exchangeratesapi.io requires an API Key

### ISSUE THREAD

**Issue by ilkohoffmann** [spiral-project/ihatemoney#737]
Requesting / in the browser results in following error:

```
ERROR [ihatemoney.run] Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask_restful/__init__.py", line 272, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/ihatemoney/ihatemoney/web.py", line 256, in home
    project_form = ProjectForm()
  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/ihatemoney/ihatemoney/forms.py", line 116, in __init__
    for currency_name in self.currency_helper.get_currencies()
  File "/ihatemoney/ihatemoney/currency_convertor.py", line 31, in get_currencies
    for rate in self.get_rates()
  File "/usr/local/lib/python3.9/site-packages/cachetools/decorators.py", line 22, in wrapper
    v = func(*args, **kwargs)
  File "/ihatemoney/ihatemoney/currency_convertor.py", line 24, in get_rates
    rates = requests.get(self.api_url).json()["rates"]
KeyError: 'rates'
```

Seems like the API http://api.exchangeratesapi.io, used by the CurrencyConverter, is not allowed to be consumed without an API Key (anymore?).

**Comment by ToadCast** [comment]
It seems to be fixed by https://github.com/spiral-project/ihatemoney/pull/736
It worked on my server.

**Comment by almet** [comment]
I confirm that I fixed this in #736. Let us know if that's not the case, thanks!
